helm/tenant/templates/NOTES.txt

```diff
@@ -2,11 +2,11 @@
   To connect to the {{ .name }} tenant if it doesn't have a service exposed, you can port-forward to it by running:
   {{- if dig "certificate" "requestAutoCert" false . }}
 
-  kubectl --namespace {{ $.Release.Namespace }} port-forward svc/{{ .name }}-console 9443:9443
+  kubectl --namespace {{ $.Values.namespaceOverride | default $.Release.Namespace }} port-forward svc/{{ .name }}-console 9443:9443
```

Comment:
Define a template plz.

Reply:
Done.

(NB: I probably forgot to push yesterday)